include/gluon.mk

```diff
@@ -35,9 +35,12 @@ $(GLUON_SITEDIR)/site.mk:
 -include $(GLUON_SITEDIR)/site.mk
 
 
-GLUON_VERSION := $(shell cd $(GLUONDIR) && git describe --always 2>/dev/null || echo unknown)
+GLUON_VERSION := $(shell cd $(GLUONDIR) && git describe --always --dirty=+ 2>/dev/null || echo unknown)
 export GLUON_VERSION
 
+GLUON_SITE_VERSION := $(shell cd $(GLUON_SITEDIR) && git describe --always --dirty=+ 2>/dev/null || echo unknown)
```

Comment:
Add `--git-dir=.git` between `git` and `describe` to ensure the command fails when GLUON_SITEDIR is no git repository.